python/tester/upload.py

```diff
@@ -1,11 +1,15 @@
+__all__ = ["get_last_group", ]
+
+import boto3
```

Comment:
What functionality is missing from lsst.resources that would make it work for this? Flexibility of endpoint (GCS vs S3 vs WebDAV) is what lsst.resources was designed to help with so it would help me if I knew why you had decided against using it. Thanks.

Reply:
This particular script is not supposed to depend on anything LSST. It was previously only useful in an environment that didn't have the Stack installed, and may be so again in the future.

Reply:
But you are installing boto3 so you are installing external software. Why can't you install lsst-resources from PyPI as well? Or are you using conda-forge so want me to add lsst-resources to conda-forge instead? lsst-resources is completely standalone PyPI installable with BSD license.

Reply:
`lsst.resources` is how butler datastore can work on S3, WebDAV, GCS, or local file system without butler having to care where the files end up.